.github/workflows/release.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-18.04, macos-10.15, windows-2019]
+        os: [ubuntu-18.04, macos-11, windows-2019]
```

Comment:
OSX 12 has been out for a year now - should we just switch to that?

Reply:
macos-11 is still what they use for macos-latest, so I figured it was the most recommended one to go to for now